docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md

```diff
@@ -48,7 +48,7 @@ To get AWS keys for S3 access:
 For more info, see
 [AWS official documentation.](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
 
-#### Google Cloud Storage credentials
+#### Google Cloud Storage / Google Drive credentials
```

Comment:
Google has unified credentials system for all their services, so it supposed to be all the same

Reply:
makes sense